SpliceRegion.pm

```diff
@@ -134,12 +141,29 @@ sub run {
         #   G => '_to_purine',
         # }
       },
-    );
+    ) unless $extended_flag;
```

Comment:
Fine but we are more used to the classic "if/else" syntax and I think Core tends to get rid of the "unless" syntax when possible.

Reply:
Fair enough, I'll keep that in mind.